scaaml/capture/input_generators/attack_point_iterator_models.py

```diff
@@ -341,3 +342,9 @@ def __len__(self) -> int:
 
     def items(self) -> AttackPointIteratorT:
         return self.iterator_model.items()
+
+
+def build_attack_points_iterator(
+        configuration: Dict[str, Any]) -> IteratorModel:
+    configuration = copy.deepcopy(configuration)
```

Comment:
NIT Probably we do not need to copy the configuration. Since Pydantic is unlikely to change it. Please check. Leaving it is also ok.
```suggestion
```

Reply:
Does not look like it